docs/docs/event-history/README.md

```diff
@@ -8,7 +8,7 @@ This includes events in progress for real-time visibility into the health of you
 
 ::: tip This feature is in beta
 
-Event Histories are currently in **beta** and are only available to Paid workspaces and with a limited number of events to start.
+Event Histories are currently in **beta** with a limited number of events to start.
```

Comment:
```suggestion
Event histories are currently in **beta** with a limited number of events to start.
```

Reply:
Changed "Histories" to "histories" to follow Google's style guide on capitalization.